src/auth/components/game-img/game-img.vue

```diff
@@ -0,0 +1,108 @@
+import { ThemeStore } from '../../../_common/theme/theme.store';
+<template>
+	<div id="cover-img-credits">
```

Comment:
don't use ID. always use classes. the standard I use for these is that the top-level div would be the name of the component in kebab-case, so `class="game-cover-credits`.

Reply:
put a `v-if` on this div to make sure `coverGame` exists: `v-if="coverGame"`